### ISSUE TITLE
PATCH HTTP method is not supported by server

### ISSUE THREAD

**Issue by kormanowsky** [dfleury2/beauty#37]
Please add support for PATCH http method

**Comment by Romain-P** [comment]
According to https://github.com/dfleury2/beauty/issues/24
You could do something like... 

```cpp
            beauty::client client;
            beauty::request request;
            request.method(beast::http::verb::patch);

            client.send_request(request, ...);
```

**Comment by kormanowsky** [comment]
> According to #24 You could do something like...
> 
> ```c++
>             beauty::client client;
>             beauty::request request;
>             request.method(beast::http::verb::patch);
> 
>             client.send_request(request, ...);
> ```

That is true for the client. But the server does not support it. I didn’t mention this, though. Updated the issue 🤝

**Comment by dfleury2** [comment]
Hi,
can you try this : https://github.com/dfleury2/beauty/tree/feature/add_patch_support
I have added PATCH support in the server in this branch
Regards,

**Comment by kormanowsky** [comment]
Hi, 
this is working for me, thanks
